src/components/UsersInfo/UsersInfo.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import { sendRequest } from '../../sendRequest/sendRequest';
+import { getUsers } from '../../sendRequest/apis';
+import { withRouter } from 'react-router-dom';
+import './UsersInfo.css';
+
+const UsersInfo = (props) => {
+
+    const [usersInfoState, setUsersInfoState] = useState({
+        usernames: [],
+        admin: '',
+    })
+
+    useEffect(() => {
+        let jwt = localStorage.getItem('jwt-eimmigrate');
+        if (!jwt) {
+            return props.history.push('/login');
+        }
+        else {
+            let requestObj = {
+                url: getUsers,
+                headers: {
+                    Authorization: `Bearer ${jwt}`
+                }
+            }
+            sendRequest(requestObj).then((usersInfo) => {
+                setUsersInfoState({
+                    usernames: usersInfo.users,
+                    admin: usersInfo.admin.name
+                })
+            })
+        }
+    });
+
+    const usernames = usersInfoState.usernames.map(user =>
+        <li key={user._id}>
+            <div className="user-name">Name: {user.name}</div>
+            <div className="phone-number">Phone number: {user.phoneNumber}</div>
+            <div className="docs">Document: {user.document}</div>
+        </li>
+    )
+    const admin = usersInfoState.admin;
+    return (
+        <div>
+            <div className="users-info">
+                Hello {admin}!
+                Our users are:
+            </div>
+            <div className="user-list">
+                <ol>
```

Comment:
this will not be a list of admin users but a list of peop;e who submitted responses to the questionair.

Reply:
it's acutally a list of users obtained from users database that was created by Nick. The admins' information is in the admins database created by me.